features/gba-dev/onbekend_waardes_gba.feature

```diff
@@ -0,0 +1,563 @@
+    | 'kind'    | kinderen      | geboorte                    | plaats | geboorteplaats (03.20)                                              | 0000   |
+    | 'kind'    | kinderen      | geboorte                    | land   | geboorteland (03.30)                                                | 0000   |
+
+  Scenario: onbekend waarde voor nationaliteit
+    Gegeven de persoon met burgerservicenummer '000000280' heeft een 'nationaliteit' met de volgende gegevens
+    | naam                            | waarde   |
+    | nationaliteit (05.10)           | 0000     |
+    | reden opname (63.10)            | 311      |
+    | datum ingang geldigheid (85.10) | 20030417 |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam                | waarde                              |
+    | type                | RaadpleegMetBurgerservicenummer     |
+    | burgerservicenummer | 000000280                           |
+    | fields              | burgerservicenummer,nationaliteiten |
+    Dan heeft de response een persoon met de volgende gegevens
+    | naam                | waarde    |
+    | burgerservicenummer | 000000280 |
+    En heeft de persoon een 'nationaliteit' met de volgende gegevens
+    | naam                              | waarde                               |
+    | nationaliteit.code                | 0000                                 |
+    | nationaliteit.omschrijving        | Onbekend                             |
+    | redenOpname.code                  | 311                                  |
+    | redenOpname.omschrijving          | Vaststelling onbekende nationaliteit |
+    | datumIngangGeldigheid             | 20030417                             |
+
+  Scenario: onbekend waarde voor reden opname nationaliteit
+    Gegeven de persoon met burgerservicenummer '000000280' heeft een 'nationaliteit' met de volgende gegevens
+    | naam                            | waarde   |
+    | nationaliteit (05.10)           | 0052     |
+    | reden opname (63.10)            | 000      |
+    | datum ingang geldigheid (85.10) | 20030417 |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam                | waarde                              |
+    | type                | RaadpleegMetBurgerservicenummer     |
+    | burgerservicenummer | 000000280                           |
+    | fields              | burgerservicenummer,nationaliteiten |
+    Dan heeft de response een persoon met de volgende gegevens
+    | naam                | waarde    |
+    | burgerservicenummer | 000000280 |
+    En heeft de persoon een 'nationaliteit' met de volgende gegevens
+    | naam                              | waarde    |
+    | nationaliteit.code                | 0052      |
+    | nationaliteit.omschrijving        | Belgische |
+    | redenOpname.code                  | 000       |
```

Comment:
Klopt dit? Volgens de Rule in regel 12 moet deze volgens mij juist niet opgenomen worden.

Reply:
De rule op 127 aangepast. Deze onbekend-waardes moeten inclusief omschrijving geleverd worden.

Bij rule 10 staat dat die waarden ongewijzigd geleverd worden. Voor de GBA-API klopt dat

Reply:
Correctie van regel 127 komt tegemoet aan mijn opmerking. Rule in regel 10 is inderdaad correct.

Reply:
Corrigeer nog wel even de spelfout in regel 127.